### PR TITLE
include information about https for images in documentation

### PR DESCRIPTION
**Pull request by SimonKenyonShepard**
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request:

> **Unless you are a React Native release maintainer and cherry-picking an _existing_ commit into a current release, ensure your pull request is targeting the `master` React Native branch.**

Explain the **motivation** for making this change. What existing problem does the pull request solve?

Prefer **small pull requests**. These are much easier to review and more likely to get merged. Make sure the PR does only one thing, otherwise please split it.

**Test plan (required)**

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Make sure tests pass on both Travis and Circle CI.

**Code formatting**

Look around. Match the style of the rest of the codebase. See also the simple [style guide](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#style-guide).

For more info, see the ["Pull Requests" section of our "Contributing" guidelines](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#pull-requests).

### CODE REVIEW

**Comment by hramos** [comment]
@SimonKenyonShepard can you fill out the issue template? It'd be useful to have the additional context on why this change is required.

File: docs/Images.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Note that this approach provides no safety checks. It's up to you to guarantee t
 
 ## Network Images
 
-Many of the images you will display in your app will not be available at compile time, or you will want to load some dynamically to keep the binary size down. Unlike with static resources, *you will need to manually specify the dimensions of your image.*
+Many of the images you will display in your app will not be available at compile time, or you will want to load some dynamically to keep the binary size down. Unlike with static resources, *you will need to manually specify the dimensions of your image and you will need to serve your image over an https connection for security.*
```
Comment (SimonKenyonShepard) [inline docs/Images.md]: it's mandated, when you try to use http it fails silently but works with https. I guess actually an improvement on fixing the documentation would also be to make it fail loudly when you try to use http.

**Comment by SimonKenyonShepard** [comment]
@hramos yeah sorry, I thought I actually did this but somehow it got lost in the process.
How should I do it now, just paste it here or something else? Also I'm unsure as why the build is failing. All I did was alter the docu.

**Comment by hramos** [comment]
It might be a flaky test, a simple docs change like this shouldn't cause these tests to fail.

You should be able to edit your original message, but if you can't, it's fine to just add a follow-up comment here (e.g. like you did when responding to Joel). Basically, some context on why https is required, what happens if it is not used, etc.

**Comment by hramos** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**